motoman_gp4_support/launch/load_gp4.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
+  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find motoman_gp4_support)/urdf/gp4.xacro'" />
```

Comment:
```suggestion
  <param name="robot_description" command="$(find xacro)/xacro '$(find motoman_gp4_support)/urdf/gp4.xacro'" />
```
inorder was only needed on ROS `indigo`

Reply:
@gavanderhoorn Can you confirm that we no longer need the --inorder, or are we keeping for backward compatibility to indigo?

Reply:
We don't support Indigo any more, but Kinetic is in extended support.

We can keep the `--inorder` for now.